teflo_init/execute/teflo.cfg

```diff
@@ -0,0 +1,4 @@
+[defaults]
+log_level=info
+workspace=.
+data_folder=.teflo
```

Comment:
This should be more detailed, you should include how users would use **credentials, timeout, concurrency settings and etc**

Reply:
@JunqiZhang0 for this example I do not think ther eis a need to add the details, because there are other examples if the user has to look for how to set up the credentials. Here we are just giving them basic workspace with a running local example which does not need credentials as such. Maybe we can always add a note to the reAdme , "for more examples checke out the examples repo"

Reply:
I added a note at the end of the README files "for more examples check out the examples repo" with a link to the examples repo.